actors.md

```diff
@@ -4,6 +4,42 @@ Any implementations of the Filecoin actors must be exactly byte for byte compati
 
 This spec decsribes a set of actors that operate within the [Filecoin State Machine](state-machine.md). All types are defined in [the basic type encoding spec](data-structures.md#basic-type-encodings).
 
+## Init Actor
+The init actor is responsible for creating new actors on the filecoin network. This is a built-in actor and cannot be replicated. In the future, this actor will be responsible for loading new code into the system (for user programmable actors).
+
+```go
+type InitActor struct {
+    NextActorID uint64
+}
+```
+
+#### Exec
+
+Parameters:
+
+- code Cid
+- params []byte
+
+TODO: the type of params should probably just be variadic set of arguments. Making it be a `[]byte` here kinda sucks
```

Comment:
@whyrusleeping depends on the level this spec is written, on a low level this has to be `[]byte` serialized in some form to pass through variadic arguments. But if this is meant to describe a high level I would agree

Reply:
Yeah, I think this can be a little higher level, how things get encoded precisely is discussed elsewhere. But this document should match closely the actual actors that get written.

Reply:
done